### DISCUSSION TITLE
Crash chrome when upload multi files using FileReader API

### DISCUSSION THREAD

**Discussion by sonphambk** [mdn/mdn-community#603] [Content]
Hi.
I tried to upload multi files using FileReader. My data includes 50 files and a file about 3MB. But when I uploaded multi files at the same time. FileReader.readAsArrayBuffer crashes the browser when I reading files
Can you help me this issue?
Thanks.

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
